frontend/i18n/en.json

```diff
@@ -769,7 +775,12 @@
     "password": "Password",
     "choose_password": "Choose a password",
     "confirm_password": "Confirm your password",
-    "please_agree": ["I agree to the", "Terms and Conditions", "and", "Privacy Policy"],
+    "please_agree": [
+      "I agree to the",
+      "Terms and Conditions",
```

Comment:
This is outdated. `staging` has different words (for example, "Term and Conditions" is now "T&C").

I guess you will get the correct strings after updating your branch from `staging`.

I suggest that you update from staging **before you solve the conflicts** otherwise you might erase the new strings.